huaweicloud/services/ga/resource_huaweicloud_ga_address_group.go

```diff
@@ -52,14 +57,13 @@ func ResourceIpAddressGroup() *schema.Resource {
 			"ip_addresses": {
 				Type:     schema.TypeSet,
 				Optional: true,
-				Computed: true,
-				ForceNew: true,
 				Elem:     ipAddressGroupSchema(),
-				MaxItems: 20,
 			},
-			"id": {
-				Type:     schema.TypeString,
-				Computed: true,
+			"listeners": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MinItems: 1,
```

Comment:
the MinItems equals required, please delete it.

Reply:
done